src/Compilers/CSharp/Portable/Syntax/LambdaUtilities.cs

```diff
@@ -47,6 +47,15 @@ public static bool IsLambda(SyntaxNode node)
         public static SyntaxNode GetLambda(SyntaxNode lambdaBody)
         {
             var lambda = lambdaBody.Parent;
+            if (lambda.Kind() == SyntaxKind.ArrowExpressionClause)
```

Comment:
This needs to be consistent across lambdas, local functions, anonymous methods, query lambdas.

---
In reply to: [134629057](https://github.com/dotnet/roslyn/pull/21599#discussion_r134629057) [](ancestors = 134629057,134613176)